src/snowflake/snowpark/udf.py

```diff
@@ -785,6 +788,28 @@ def _do_register_udf(
                 "You cannot create a non-vectorized UDF using pandas_udf(). "
                 "Use udf() instead."
             )
+        elif is_pandas_udf and isinstance(func, tuple):
+            # Checks if the file explicitly marks the handler function as vectorized, otherwise copy the file and append <handler>._sf_vectorized_input = pandas.DataFrame
+            vectorized_udf_specification = [
+                "\n@vectorized",
+                f"\n{func[1]}._sf_vectorized_input =",
+            ]
+            file_content = open(func[0]).read()
+            if not any(spec in file_content for spec in vectorized_udf_specification):
```

Comment:
Will this be kinda fragile if user has, say, a triple-quoted string that has the same content? Is using `ast` to understand the file an overkill?